website/docs/how-to/define-api-in-typescript.md

```diff
@@ -47,7 +47,7 @@ First, you'll need to [install the proper dependencies](../getting-started/guide
 ```swift title=EchoPlugin.swift
 import Capacitor
 
-@objc(ECHO_PLUGIN)
+@objc(MYYEchoPlugin)
```

Comment:
Is this supposed to by MYEchoPlugin?

Reply:
The convention is to use a 3 letter prefix in objective-c. I didn't want to encourage people to use ION or CAP 😅
I'm open to other suggestions

Reply:
That's fine I just didn't know if it was a typo